packages/provider_devtools_extension/lib/src/provider_screen.dart

```diff
@@ -110,10 +111,15 @@ class ProviderScreenBody extends ConsumerWidget {
 }
 
 void showProviderErrorBanner() {
-  // TODO(kenz) post message here
-  // bannerMessages.addMessage(
-  //   ProviderUnknownErrorBanner(screenId: ProviderScreen.id).build(),
-  // );
+  extensionManager.showBannerMessage(
+    key: 'provider_unknown_error',
+    type: 'error',
+    message: '''
+DevTools failed to connect with package:provider.
+
+This could be caused by an older version of package:provider; please make sure that you are using version >=5.0.0.''',
```

Comment:
Isn't the devtool version tied with the provider package version?

In which case, shouldn't they already be using the "up-to-date" version?

Reply:
This was just copied over from what the existing provider screen in DevTools had. Now that the provider screen (as a DevTools extension) is tied to the package:provider version is is shipped with, this is probably obsolete.